tests/test_index.py

```diff
@@ -362,3 +365,33 @@ def test_linear_index_moltype_select():
     # select something impossible
     linear2 = linear.select(ksize=4)
     assert len(linear2) == 0
+
+
+@utils.in_tempdir
+def test_index_same_md5sum_fsstorage(c):
+    testdata_glob = utils.get_test_data('img/*.sig')
```

Comment:
I'm a little against wildcards here, because it makes it possible that we may add a new sig file and break the test for other reasons.

Reply:
fixed